### DISCUSSION TITLE
how to build a BIN file from a python project ?

### DISCUSSION THREAD

**Discussion by ds2k5** [pyinstaller/pyinstaller#8861] [Help]
Hi,
found this project:  https://github.com/C0D3D3V/bbb-dl

Like to be able to build (upx compressed) for: Linux, MacOS and Windows a static bin file.

Needs to include: ffmpeg and playwright

How is this possible ?

**Comment by bwoodsend** [comment]
I'm sure you can read the documentation yourself.
